lightblue/src/main/java/org/esbtools/eventhandler/lightblue/LightblueNotificationRepository.java

```diff
@@ -80,10 +78,14 @@ public LightblueNotificationRepository(LightblueClient lightblue, String[] entit
             DataBulkRequest updateEntities = new DataBulkRequest();
```

Comment:
Similarly: in markNotificationsProcessedOrFailed: a bulk request with two
updates, one for processed, and one for failed, would perform better.

On Mon, Jan 25, 2016 at 10:31 AM, Alec Henninger <notifications@github.com>
wrote:

> In
> lightblue/src/main/java/org/esbtools/eventhandler/lightblue/LightblueNotificationRepository.java
> <https://github.com/esbtools/event-handler/pull/8#discussion_r50726381>:
>
> > @@ -80,10 +78,14 @@ public LightblueNotificationRepository(LightblueClient lightblue, String[] entit
> >              DataBulkRequest updateEntities = new DataBulkRequest();
>
> Added #11 <https://github.com/esbtools/event-handler/issues/11>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/esbtools/event-handler/pull/8/files#r50726381>.
>